### ISSUE TITLE
Cycleways overlay - Options to map - Clarification possible?

### ISSUE THREAD

**Issue by Filbuntu** [streetcomplete/StreetComplete#5659]
**Use case**
Mapping a foot trail (bicycle not allowed) seems a bit tricky to me. I used the cycling overlay and you can not choose "Cycling not allowed". There is an option "Not designated for cyclist (but may be allowed)." which suits not, either. That is why I chose "Path or Trail" and I thought this would mean the trail is only a foot path. But when I checked this turned on "Bicycle" and "Foot" with "yes" 🫣.

**Proposed Solution**
Would it be possible to add (Foot & bicycle allowed) to the option "Path or Trail"? E.g Path or Trail (Foot & bicycle allowed).
Would it be possible to add an option "Foot path only (bicycle not allowed)"? Or is there a reason not to have this option?

Thanks!

**Comment by westnordost** [comment]
What makes that foot trail a foot trail? Is there a sign?

**Comment by Filbuntu** [comment]
Yes, it is an obvious foot trail and it states that it is for pedestrians and bicycles are not allowed.

**Comment by westnordost** [comment]
Right, good. So, 

>  Or is there a reason not to have this option?

Yes, there is a reason to not have this option. See #4913

However, for the case of that there is actually a sign either prohibiting or allowing bikes explicitly, a solution is being worked on in #5575, in fact, it is basically complete.

**Comment by westnordost** [comment]
Hence, I will close this as duplicate of #4913

**Comment by westnordost** [comment]
That's an early screenshot. There'll be an option for "footway, cycling explicitly forbidden"

**Comment by Filbuntu** [comment]
Great! Thanks in advance for adding this option!
For more clarity: Would it be possible to add (Foot & bicycle allowed) to the option "Path or Trail"? E.g Path or Trail (Foot & bicycle allowed).
If you come up with a better wording, this fine, too 😊.

**Comment by westnordost** [comment]
Only because it is a path does not necessarily mean both foot and bicycle are allowed. May depend on the local legislation. "Path or trail" is `highway=path`, nothing more, nothing less. 

If it is explicitly signed to be a footway, there is that option. If it is explicitly signed to either allow or prohibit bicycles, there are those other (soon to be added) options.

**Comment by Filbuntu** [comment]
Thank you for the clarification. It seems more complicated than I thought 😊.
In my situation, I mapped a trail as "Path or Trail" in the StreetComplete app (it was "red" (not mapped) in the cycleways overlay) and it was tagged as highway=path (from the previous tagging highway=footway) and bicycle=yes (from bicycle=no) which I discovered when I checked on OSM. 
Could the tagging highway=path trigger different other tagging (like bicycle, foot) depending on the specific country?

**Comment by westnordost** [comment]
Country-specific tagging behavior that are not based on actually different on-site situations but just local tagging practice preference would be too complex to implement and maintain.

**Comment by westnordost** [comment]
Not sure why `highway=footway` + `bicycle=no` was marked red before, though. That would be a bug. But, well, no point in investigating, at least not until it can be reproduced after #5575 is merged and released.

**Comment by Filbuntu** [comment]
Thank you for your helpful clarifications! 
With the new options there should be no problem anymore.
Keep up the excellent work!
